src/com/gh4a/activities/SearchActivity.java

```diff
@@ -155,6 +186,24 @@ public void onCreate(Bundle savedInstanceState) {
         mSearch.setOnCloseListener(this);
         mSearch.onActionViewExpanded();
 
+        mSearch.setOnSuggestionListener(this);
+        mSearch.setSuggestionsAdapter(
+                new SimpleCursorAdapter(this, android.R.layout.simple_list_item_1, null,
+                        new String[]{SuggestionsProvider.Columns.SUGGESTION},
+                        new int[]{android.R.id.text1}, 0)
+        );
+
+        final SearchView.SearchAutoComplete complete = (SearchView.SearchAutoComplete) mSearch.findViewById(R.id.search_src_text);
+        complete.setThreshold(0);
+        complete.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                complete.showDropDown();
+            }
+        });
```

Comment:
This is required to show suggestions when user clicks on `SearchView`. But this code doesn't make sense if suggestions are shown after 2 symbols.